content/6_Advanced/Extend_Euclid.mdx

```diff
@@ -129,6 +204,19 @@ pl extendEuclid(ll a, ll b) { // returns {x,y}
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+
+def extend_euclid(a: int, b: int) -> list[int]:
+    if not b:
+        return [1, 0]
+    p = ext_euclid(b, a%b)
+    return [p[1], p[0] - (a//b)*p[1]]
```

Comment:
```suggestion
    return [p[1], p[0] - (a // b) * p[1]]
```

Reply:
if sans or envy told you to do otherwise, ignore this

Reply:
See: https://peps.python.org/pep-0008/#other-recommendations

Reply:
https://usaco.guide/general/adding-solution?lang=py#code-conventions

we don't follow pip strictly, although if sans or envy say otherwise, then its okay.

Reply:
the original code is fine as is